### DISCUSSION TITLE
RAW Photo Deletion // Immich-Go Uploading Syntax Question

### DISCUSSION THREAD

**Discussion by lovelacecreativeco** [simulot/immich-go#126] [Q&A]
For some reason, when importing my takeout path it still uploaded all of my raw images. 

Here is my command: 

./immich-go -server=https://{url here} -key={key here} upload -create-albums -google-photos -keep-partner=false -exclude-types .dng,.cr2,.arw,.rw2,.tif,.tiff,.gif,.psd "D:\Takeout\takeout-*.zip"

Are the exclude-types case sensitive? My Takeout Archive was 5TB so I really don't want to have to cancel processing or start over since it took about 12 hours just for Immich Go to go through everything and actually start the upload process.

Now I need to go through and delete all RAW images from the library, since I really don't want those to be taking up storage twice since I backup all RAW images separately.

Is there a tool within immich-go or the Immich CLI that anyone knows of to delete all RAW files from the Immich library (and storage)?

Thanks!

**Comment by simulot** [comment]
Confirmed. this is a regression.

No, I don't know how to remove raw file in the correct way.  But this could be a feature for immich-go.

**Reply by lovelacecreativeco** [reply]
Thanks for fixing the -exclude-types issue!

What are the odds a remove RAW files or remove files with X, Y, Z extension (similar to -exclude-types syntax) could be implemented?

Thanks for your hard work on this!

**Reply by simulot** [reply]
this, is on my plans... something like immich-go tool delete assets -daterange xxxxx *.raw

**Reply by sayyedsuboor** [reply]
But whats the way to use exclude filetype. The below command gives an error
./immich-go -server=http://192.168.xxx.xxx:2283 -key=p0y5xzphVebKSxv2qGhXr0nnqz9KMbNhypOsMPxxxx upload -create-albums *.*

gives an error 
files with extension '.bmp'.'tnfp'.'lmp' are not supported. Check the discussion here https://github.com/simulot/immich-go/discussions/109

and discussion 109 does no throw any light on any of these

How to use exclude file types????

**Reply by simulot** [reply]
Immich-go (as of version 0.11.0) get the list of accepted file extension from the server  [itself](https://github.com/immich-app/immich/blob/8ce18b3403d286862275fd5be9cc3984fb9bf89a/server/src/domain/domain.constant.ts#L96-L162)

.bmp is supposed to be accepted.
